src/utils/whitelistApis.ts

```diff
@@ -3567,6 +3567,13 @@ export const API_LIST = {
                         ROLE.PUBLIC,
                    ],
         },
+        '/proxies/v8/designationsUpload/upload': {
```

Comment:
Update the URL to -> '/proxies/v8/designation/upload'
use the same in the below

Reply:
made the changes